.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "common"]
         path = common
-        url = https://gitlab.freedesktop.org/gstreamer/common.git
+        url = git@github.com:fluendo/common.git
```

Comment:
if we're changing the repo of submodule, we also need to update the hash, isn't it? Otherwise it won't fetch

Reply:
This should be already be pointing to the correct hash in our common repo now, it should be the branch with the same name.

Reply:
you're right, I've checked that with git clone, well done!